app/lib/widgets/wallets/add_wallet.dart

```diff
@@ -138,6 +138,11 @@ class _NewWalletState extends State<NewWallet> {
       return false;
     }
 
+    if (walletSecret.startsWith('S') && widget.wallets.any((wallet) => wallet.stellarSecret == walletSecret)){
```

Comment:
please do the same for the tfchain wallets

Reply:
done